ballista/scheduler/src/state/task_manager.rs

```diff
@@ -110,8 +107,14 @@ impl ActiveJobQueue {
     }
 
     pub fn push(&self, job_id: String, graph: ExecutionGraph) {
+        let running_stage = graph.running_stages().len();
```

Comment:
@thinkharderdev what would be the correct number here?

Reply:
I think we probably want the max running stage so 
```
graph.running_stages().iter().max().unwrap_or_default()
```